atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -183,6 +183,7 @@ long grabLocksAndRun(SweepLocks locks) throws InterruptedException {
                 if (locks.haveLocks()) {
                     sweptSuccessfully = runOnce();
                 } else {
+                    sweepMetrics.unableToAcquirePersistedLock();
```

Comment:
Ah. Which one is the one that is stored in `_persisted_locks` that needs to be cleared if a sweep is killed ungracefully? Main thing I want to expose here is if/when we're in a state that potentially needs manual intervention.